exampleSite/content/about/pricing/plan-1.md

```diff
@@ -0,0 +1,20 @@
++++
+weight = 10
+
+title = "Starting plan"
+subtitle = "starting at"
+
+price = "Free"
+# highlight = true
+
+button_text = "Start for free"
+button_url = "#"
+
+[[feature]]
+  text = "**Basic** feature"
+  icon = "fas fa-check"
+
+[[feature]]
```

Comment:
We do plural but I based it on `menu` in `config.toml`. Seems Hugo is the one with problems since it has `params`, `taxonomies` and `author` and `menu`. I'll do plural that makes sense.